qadence/ml_tools/saveload.py

```diff
@@ -14,20 +14,25 @@
 logger = getLogger(__name__)
 
 
-def get_latest_checkpoint_name(folder: Path, type: str) -> Path:
+def get_latest_checkpoint_name(folder: Path, type: str, device: str | torch.device = "cpu") -> Path:
```

Comment:
I think if we're to allow `torch.device` here we should also perform the same transformation as we do for writing it
```
str(model.device).split(":")[0]
```

Reply:
Yes true!